purchases/src/main/kotlin/com/revenuecat/purchases/PurchasesException.kt

```diff
@@ -0,0 +1,13 @@
+package com.revenuecat.purchases
+
+class PurchasesException(val error: PurchasesError) : Exception() {
```

Comment:
We should also add API tests for this

Reply:
There are API tests for this class already https://github.com/RevenueCat/purchases-android/pull/1012/files#diff-6683d7486ea14232e0f84ed70f3a7dca6bc3c3d29eaf1232bcdacde8f5b08fef

Reply:
🤦 Yup, ignore me 😅